### PR TITLE
Set up repository

### PR DESCRIPTION
**Pull request by BramvdnHeuvel**
This pull request contains some files that allow the programmer to make some unittests, effectively testing the code super quickly.
A file `requirements.txt` has also been added to help a new user gain the packages they need for the Python program to work.

### CODE REVIEW

**Comment by BramvdnHeuvel** [comment]
I have added a `.gitignore` file. This is for the purpose of preventing irrelevant files from popping up as potential commits all the time. Very annoying.

**Review by runeforger44 — APPROVED** [review]
Toppie!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
+  - "nightly"
+# command to install dependencies
```
Comment (schipeltaper) [inline .travis.yml:5]: Niet mee eens

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
```
Comment (theyearis1981) [inline .travis.yml:3]: Ouderwets >:(

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
+  - "nightly"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```
Comment (theyearis1981) [inline .travis.yml:7]: Wie is pip?????????

**Review by schipeltaper — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (edgar-right) [inline .travis.yml:5]: nah

File: test.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import unittest
```
Comment (theyearis1981) [inline test.py:1]: bliep

**Review by edgar-right — COMMENTED** [review]
han

**Review by theyearis1981 — APPROVED** [review]

**Review by edgar-right — APPROVED** [review]

**Pull request merged**